src/provideInlineCompletionItems.ts

```diff
@@ -34,11 +34,7 @@ export default async function provideInlineCompletionItems(
       return await getLookAheadSuggestion(document, completionInfo, position);
     }
 
-    const completions = await tryGetDebouncedCompletions(
-      document,
-      position,
-      token
-    );
+    const completions = await debounceCompletions(document, position, token);
```

Comment:
i see you haven't generalized the debounce function - not a blocker for this pr, just making sure you haven't forgotten it

Reply:
see my comment [here](https://github.com/codota/tabnine-vscode/pull/993#discussion_r1057174733)  bc of the unique way of calculating the debounce value (debounceMs - time of current query) and that i do want to re fetch completions after the debounce, i do not think there is a real value of extracting it, it will converts this function  to less readable